### DISCUSSION TITLE
Pre-release should be created with `prerelease: true`

### DISCUSSION THREAD

**Discussion by budimanjojo** [fluxcd/flux2#3776] [Proposals] [answered]
The latest `v2.0.0-rc.1` is being released as a normal release instead of a `prerelease`. The affect is https://github.com/fluxcd/flux2/releases/latest will point to a release candidate. Can this be avoided in the next RC? Thank you.

**Comment by makkes** [comment, marked as answer]
Releases on the fluxcd/flux2 repository have never been marked as pre-releases and as noted in [fluxcd/flux2/issues/3782](https://github.com/fluxcd/flux2/issues/3782#issuecomment-1501802910), everyone on a 0.x release should upgrade to v2.0.0-rc.1.

**Comment by budimanjojo** [comment]
Okay thank you
